### ISSUE TITLE
Network request

### ISSUE THREAD

**Issue by barnesjt** [Barnesjt/appstudio-weather#4]
Need to be able to send a network request to a weather API to get 5-10 days of weather info.

**Comment by barnesjt** [comment]
Relevant documentation:
https://developers.arcgis.com/appstudio/api-reference/qml-arcgis-appframework-networkrequest

Code written during yesterday's exploration:
```
    NetworkRequest {
      id: weatherRequest
      url: urlFromCoords(weatherDB.longitude, weatherDB.latitude)
      onReadyStateChanged: {
        if(readyState == NetworkRequest.DONE){
          if(errorCode !== 0) {
            getDBRecent();
          } else {
            putDB(Date.now(), weatherDB.latitude, weatherDB.longitude, responseText);
            getDBRecent();
          }
        }
      }
    }
```
And then in this case, the request can be sent with `weatherRequest.send();`. urlFromCoords generates the request URL from the latitude and longitude inputs. The putDB and getDBRecent are function to update the database, which is covered in #5 .

**Comment by barnesjt** [comment]
Resolved in PR #9
